src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -301,8 +308,7 @@ require.config({
       lightbox: false
     }, options);
     return this.each(function() {
-      var xmlDoc = mxUtils.parseXml($(this).data('model'));
-      var graphViewer = new GraphViewer(this, xmlDoc.documentElement, options);
+      var graphViewer = new GraphViewer(this,  getXMLDiagram($).documentElement, options);
```

Comment:
Passing the jQuery instance as parameter doesn't make sense. You probably wanted to pass ``$(this)``. Also, since the function returns the XML document that corresponds to the diagram (not the diagram of an XML document) then I think a better name would be ``getDiagramXMLDocument`` (e.g. ``getImageWidth`` makes more sense than ``getWidthImage``).